src/main/java/org/broadinstitute/hellbender/engine/spark/SparkCommandLineArgumentCollection.java

```diff
@@ -17,7 +18,10 @@
 public final class SparkCommandLineArgumentCollection implements Serializable {
     private static final long serialVersionUID = 1L;
 
-    @Argument(fullName = "spark-master", doc="URL of the Spark Master to submit jobs to when using the Spark pipeline runner.", optional = true)
+    public static final String SPARK_MASTER_LONG_NAME = "spark-master";
+    public static final String SPARK_VERBOSITY_LONG_NAME = "spark-verbosity";
+
+    @Argument(fullName = SPARK_MASTER_LONG_NAME, doc="URL of the Spark Master to submit jobs to when using the Spark pipeline runner.", optional = true)
```

Comment:
Here and below, I might break out by line and add horizontal white space to match the style of the `sparkProperties` argument.  Could also fix up the capitalization of `spark` there.

Reply:
Broke up the lines ala `programName` parameter in `SparkCommandLineProgram`